### PR TITLE
Add quick launch for Accessibility settings

### PR DESCRIPTION
**Pull request by hkamran80**
Found the URL on [Stack Overflow](https://stackoverflow.com/a/59120311/7313822)

### CODE REVIEW

**Comment by hkamran80** [comment]
Squirrel - Accessibility quick launch

**Comment by aheze** [comment]
Awesome, thanks!

File: Sources/Squirrel/ContentView.swift [hunk 1]
```diff
@@ -96,13 +96,20 @@ struct ContentView: View {
...
+                            .fontWeight(.semibold)
+
+                        Text("Tap to open Accessibility settings")
+                            .foregroundColor(.white.opacity(0.7))
+                            .font(.body)
+                            .fontWeight(.medium)
+                    }
+                    .padding(12)
+                    .background(Color.blue.brightness(-0.2))
+                    .frame(maxWidth: .infinity, alignment: .leading)
```
Comment (aheze) [inline Sources/Squirrel/ContentView.swift:112]: This should probably be reversed:

```swift
.frame(maxWidth: .infinity, alignment: .leading)
.background(Color.blue.brightness(-0.2))
```

Currently looks like this (seems to be slightly short):

Right side of blue background is clipped

Future note: the text color contrast could be improved

**Review by aheze — CHANGES_REQUESTED** [review]

File: Sources/Squirrel/ContentView.swift (hunk 1 above)
Comment (hkamran80) [reply Sources/Squirrel/ContentView.swift]: The contrast is fine. The background for the steps isn't showing up in the screenshots very well.

**Review by aheze — APPROVED** [review]

**Pull request merged**